units/crons.go

```diff
@@ -902,13 +907,46 @@ func PopulateHostSetupJobs(env evergreen.Environment) amboy.QueueOperation {
 			return errors.Wrap(err, "error fetching provisioning hosts")
 		}
 
-		ts := utility.RoundPartOfMinute(30).Format(TSFormat)
+		sort.Slice(hosts, func(i, j int) bool {
+			return hosts[i].StartTime.Before(hosts[j].StartTime)
+		})
+
+		attemptsUsed := 0
+		jobsSubmitted := 0
+		collisions := 0
 		catcher := grip.NewBasicCatcher()
 		for _, h := range hosts {
-			catcher.Add(queue.Put(ctx, NewHostSetupJob(env, h, ts)))
+			if !h.IsContainer() {
+				if time.Since(h.StartTime) < 40*time.Second {
```

Comment:
Any reason for 40 seconds being the magic time?

Related, but I assume it's because we're waiting for sshd but this will slow down provisioning for all hosts.

Reply:
I looked at minimum times for host provisioning in splunk and couldn't find anything that came up in less than 40 seconds. 

adding a comment to explain.